.tests/pf-logs/pf-logs.log

```diff
@@ -1,3 +1,8 @@
 Sep 28 10:38:09 pfSense filterlog: 53,,,1000000301,em0,match,block,in,4,0x0,,64,420,0,none,6,tcp,460,10.0.2.2,10.0.2.15,53742,22,420,PA,29633380:29633800,3617062173,65535,,
 Sep 28 10:38:24 pfSense filterlog: 53,,,1000102631,em0,match,block,in,4,0x0,,64,600,0,none,6,tcp,44,10.0.2.2,10.0.2.15,53792,22,0,S,31552001,,65535,,mss
-Sep 28 10:38:41 pfSense filterlog: 6,,,1000102434,em0,match,block,out,4,0x0,,64,0,0,DF,6,tcp,40,10.0.2.15,10.0.2.2,22,53742,0,FA,3617062173,29633380,65535,,
\ No newline at end of file
+Sep 28 10:38:41 pfSense filterlog: 6,,,1000102434,em0,match,block,out,4,0x0,,64,0,0,DF,6,tcp,40,10.0.2.15,10.0.2.2,22,53742,0,FA,3617062173,29633380,65535,,
```

Comment:
Are these log examples from opnsense? if so could we change the machine name from hostname to opnsense? so when checking the assert we can easily see which is opnsense and pfsense.?

Reply:
The first three lines are the original lines in that log. All I did was add a few of my OPNsense lines. This way we have a log containing a mix of pfSense and OPNsense logs. I was hoping that that would help upkeep it being compatible with both. It's just marked as changed because there was no new line before.

Reply:
Sorry meant the lines below that have `hostname` as the machine name. Would be nice if we could change them to `opnsense` then we know looking at the tests which is pfsense or opnsense

Reply:
I haven't touched this since September so lemme see if I can recall all the commands and stuff haha.

Reply:
I can do it if you like saves time

Reply:
I would *really* appreciate that. I honestly just made this to show a combined regex and was hoping someone could fully integrate it into crowdsec. :pleading_face: